### PR TITLE
triangle insertion

### PR DESCRIPTION
**Pull request by Yixin-Hu**

### CODE REVIEW

**Comment by Yixin-Hu** [comment]
@teseoch The triangle insertion works. Please fix the test and merge the PR.

File: app/tetwild/TetWild.h [hunk 1]
```diff
@@ -181,13 +183,13 @@ class TetWild : public wmtk::TetMesh
...
     //
     void add_tet_centroid(const std::array<size_t, 4>& vids) override;
-    void insertion_update_surface_tag(
-        size_t t_id,
-        size_t new_t_id,
-        int config_id,
-        int diag_config_id,
-        int index,
-        bool mark_surface) override;
+//    void insertion_update_surface_tag(
```
Comment (teseoch) [inline app/tetwild/TetWild.h:186]: delete this

File: src/wmtk/TetMesh.h [hunk 2]
```diff
@@ -326,22 +326,16 @@ class TetMesh
...
 protected:
-    virtual void insertion_update_surface_tag(
-        size_t t_id,
-        size_t new_t_id,
-        int config_id,
-        int diag_config_id,
-        int index,
-        bool mark_surface)
-    {}
     virtual void add_tet_centroid(const std::array<size_t, 4>& vids) {}
```
Comment (teseoch) [inline src/wmtk/TetMesh.h:339]: this should have tuples

File: src/wmtk/TriangleInsertionConn.cpp [hunk 3]
```diff
@@ -8,29 +8,155 @@
...
+//        }
+//        cout<<endl;
+//    }
+//    cout<<"====="<<new_face_vids.size()<<endl;
+//    for(auto& f: old_face_vids){
+//        for(int j=0;j<5;j++)
+//            cout << f[j] << " ";
+//        cout<<endl;
+//    }
+//    cout<<"====="<<old_face_vids.size()<<endl;
```
Comment (teseoch) [inline src/wmtk/TriangleInsertionConn.cpp:126]: lets delete commented code

File: src/wmtk/TetMesh.h (hunk 2 above)
Comment (Yixin-Hu) [reply src/wmtk/TetMesh.h]: changed to take tuple as input

**Pull request merged**